scripts/tunnelstat

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/env python3
+
+#####################################################################
+#
+# tunnelstat is a tool for summarizing l3 network statistics.
```

Comment:
Looks like description is not accurate, probably it should be something like "VxLAN tunnel" statistics but not "L3 network".